packages/citrix_adc/data_stream/log/elasticsearch/ingest_pipeline/native.yml

```
@@ -13,10 +13,11 @@ processors:
         - '^%{SPACE}%{HEADER_NOTIMEZONE} : %{DATA:_tmp.details} : +"%{GREEDYDATA:citrix.extended.message}"'
         - '^%{SPACE}%{HEADER} : %{DATA:_tmp.details} : +"%{GREEDYDATA:citrix.extended.message}"'
         - '^%{SPACE}%{HEADER} : %{DATA:_tmp.details} : +%{GREEDYDATA:citrix.extended.message}'
+        - '^<%{NUMBER}>%{NUMBER} (%{TIMESTAMP_ISO8601:_tmp.timestamp}|-) (%{SYSLOGHOST:citrix.host}|-) (%{DATA:_tmp.appname}|-) (%{DATA:_tmp.procid}|-) (%{DATA:_tmp.msgid}|-) (%{DATA:_tmp.structured_data}|-) (%{DATA:_tmp.details} :)?%{SPACE}"?%{GREEDYDATA:citrix.extended.message}"?$'
```

Could we make these non-capture groups?
```suggestion
        - '^<%{NUMBER}>%{NUMBER} (?:%{TIMESTAMP_ISO8601:_tmp.timestamp}|-) (?:%{SYSLOGHOST:citrix.host}|-) (?:%{DATA:_tmp.appname}|-) (?:%{DATA:_tmp.procid}|-) (?:%{DATA:_tmp.msgid}|-) (?:%{DATA:_tmp.structured_data}|-) (?:%{DATA:_tmp.details} :)?%{SPACE}"?%{GREEDYDATA:citrix.extended.message}"?$'
```

---

 changed

---

These are still showing up as capturing groups.